src/karts/kart.cpp

```diff
@@ -1890,6 +1900,29 @@ void Kart::handleMaterialGFX(float dt)
 {
     const Material *material = getMaterial();
 
+    if (m_skidding->isSkidding())
+    {
+        m_dirt_factor += 0.001f;
```

Comment:
This should depend on the dt this function is called with, so that the time to dirty is frame-rate independent.